lib/mongoid/contextual/geo_near.rb

```diff
@@ -46,6 +46,22 @@ def each
         end
       end
 
+      # Projects an output field containing the distance from each document
+      # to the point in the command.
+      #
+      # @example Specify the distance field.
+      #   geo_near.distance_field("dist")
+      #
+      # @param [ String ] value The distance field.
+      #
+      # @return [ GeoNear ] The GeoNear wrapper.
+      #
+      # @since 2.4.0
```

Comment:
I'm not sure this is correct. I'd expect the version provided by `@since` to be the Mongoid version that introduced the API. Looking at the other methods, I don't think those versions refer to server versions as this one does.

Reply:
Fair point. I did my best to look through the previous docs but I found that I could only go as far back as `2.4.0`, which is why I called it out in the PR description just in case this wasn't correct. Any help to track this down then would be appreciated.